.circleci/config.yml

```diff
@@ -78,7 +78,7 @@ commands:
           steps:
             - run:
                 name: Install pnpm package manager
-                command: corepack enable
+                command: npm i -g corepack
```

Comment:
but `corepack enable` still needs to run, right? Wouldn't it need to go in a separate step?

```diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index f43caed556..8f061b96ed 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -69,9 +69,8 @@ commands:
         description: 'Set to true if you intend to any browser (for example with playwright).'
     steps:
       - run:
-          name: Set npm registry public signing keys
-          command: |
-            echo "export COREPACK_INTEGRITY_KEYS='$(curl https://registry.npmjs.org/-/npm/v1/keys | jq -c '{npm: .keys}')'" >> $BASH_ENV
+          name: Install corepack
+          command: npm i -g corepack
 
       - when:
           condition: << parameters.browsers >>
```

Reply:
When you globally install corepack, the command `corepack enable` is not required to be run.

Reply:
Waiting out for corepack commands to fail in Node.js 25+ is an option.
Even if it's added in Docker images, the separate install command will just be redundant.

I'm just installing corepack in popular packages, so that corepack maintainers get data that there are users who will continue to use it, and it's maintained. More details in https://github.com/nodejs/corepack/issues/681

Reply:
honestly, if the docker image comes with a modern enough version of pnpm prebundled, then I don't want to use corepack at all.

Reply:
If you're okay giving control of the package manager version in this project to one installed by CI provider and enforce every contributor to use (and update) the same version in their workspace, then it should be fine.

Reply:
newer versions of pnpm read the packageManagerField and install the correct version. i.e. pnpm can already do what corepack does.